### DISCUSSION TITLE
"TypeError: Failed to fetch"

### DISCUSSION THREAD

**Discussion by Berti79** [comfyanonymous/ComfyUI#471] [Q&A]
Hello sorry for this beginner question,

I am a newcomer to AI and Linux, I managed the installation and I was able to start the programme in the browser. Then I tried to create the Deafoult image.

The following error occurred:
"TypeError: Failed to fetch".

And at the moment the UI can no longer be opened.
After reinstalling, the same thing happens!

thanks for help

**Comment by ltdrdata** [comment]
Typically, this issue occurs due to a problem with the network. Are you currently using ComfyUI on Colab? In that case, if Local Tunnel service becomes unstable intermittently, such a phenomenon may occur.

**Reply by Berti79** [reply]
No i run it on my local Notebook

**Reply by bobcanada** [reply]
There is a same problem in my comfyui. It's puzzle.

**Comment by franksmps** [comment]
me either

**Reply by ltdrdata** [reply]
You need to check the terminal.
This phenomenon also occurs when ComfyUI is killed due to issues like insufficient memory.

**Reply by franksmps** [reply]
thank u sir
